src/bridge/test-fs.c

```diff
@@ -435,6 +435,31 @@ test_read_removed (TestCase *tc,
 }
 
 static void
+test_read_non_mmappable (TestCase *tc,
+                         gconstpointer unused)
+{
+  gchar *tag;
+  JsonObject *control;
+  const gchar *path = "/sys/power/state";
```

Comment:
The test should check that this file actually exists (it migth not in a container). If it doesn't exist, then the test should ```cockpit_test_skip()``` and ```return;```